source/cli/realm-cli-reference-v1.txt

```diff
@@ -249,7 +249,7 @@ that doesn't exist, {+cli-bin+} can :ref:`create the application
    If specified, uploads and deploys any static assets in the
    ``/hosting/files`` directory of your App.
 
-.. option:: --include-dependencies
+.. option:: --include-node-modules
```

Comment:
This file is for realm-cli version 1. This is an older version of realm-cli that I do not believe is getting updates anymore. Before we change this flag in the v1 reference, we should try to install a v1 realm-cli and see if this flag has actually been updated there.

Separately, we may want to remove the v1 reference as it has now been quite a while since realm-cli v2 was released and I'm not sure how much this is needed anymore. It may cause more confusion than help at this point since there is now a much more comprehensive v2 reference.